nnpdf_data/nnpdf_data/commondata/ATLAS_Z0_8TEV_HIMASS/uncertainties.yaml

```diff
@@ -0,0 +1,1978 @@
+definitions:
+  stat:
+    description: stat
+    treatment: MULT
+    type: UNCORR
+  sys,unc:
+    description: sys,unc
+    treatment: MULT
+    type: UNCORR
+  sys,cor1:
+    description: sys,cor1
+    treatment: MULT
+    type: CORR
+  sys,cor2:
+    description: sys,cor2
+    treatment: MULT
+    type: CORR
+  sys,cor3:
+    description: sys,cor3
+    treatment: MULT
+    type: CORR
+  sys,cor4:
+    description: sys,cor4
+    treatment: MULT
+    type: CORR
+  sys,cor5:
+    description: sys,cor5
+    treatment: MULT
+    type: CORR
+  sys,cor6:
+    description: sys,cor6
+    treatment: MULT
+    type: CORR
+  sys,cor7:
+    description: sys,cor7
+    treatment: MULT
+    type: CORR
+  sys,cor8:
+    description: sys,cor8
+    treatment: MULT
+    type: CORR
+  sys,cor9:
+    description: sys,cor9
+    treatment: MULT
+    type: CORR
+  sys,cor10:
+    description: sys,cor10
+    treatment: MULT
+    type: CORR
+  sys,cor11:
+    description: sys,cor11
+    treatment: MULT
+    type: CORR
+  sys,cor12:
+    description: sys,cor12
+    treatment: MULT
+    type: CORR
+  sys,cor13:
+    description: sys,cor13
+    treatment: MULT
+    type: CORR
+  sys,cor14:
+    description: sys,cor14
+    treatment: MULT
+    type: CORR
+  sys,cor15:
+    description: sys,cor15
+    treatment: MULT
+    type: CORR
+  sys,cor16:
+    description: sys,cor16
+    treatment: MULT
+    type: CORR
+  sys,cor17:
+    description: sys,cor17
+    treatment: MULT
+    type: CORR
+  sys,cor18:
+    description: sys,cor18
+    treatment: MULT
+    type: CORR
+  sys,cor19:
+    description: sys,cor19
+    treatment: MULT
+    type: CORR
+  sys,cor20:
+    description: sys,cor20
+    treatment: MULT
+    type: CORR
+  sys,cor21:
+    description: sys,cor21
+    treatment: MULT
+    type: CORR
+  sys,cor22:
+    description: sys,cor22
+    treatment: MULT
+    type: CORR
+  sys,cor23:
+    description: sys,cor23
+    treatment: MULT
+    type: CORR
+  sys,cor24:
+    description: sys,cor24
+    treatment: MULT
+    type: CORR
+  sys,cor25:
+    description: sys,cor25
+    treatment: MULT
+    type: CORR
+  sys,cor26:
+    description: sys,cor26
+    treatment: MULT
+    type: CORR
+  sys,cor27:
+    description: sys,cor27
+    treatment: MULT
+    type: CORR
+  sys,cor28:
+    description: sys,cor28
+    treatment: MULT
+    type: CORR
+  sys,cor29:
+    description: sys,cor29
+    treatment: MULT
+    type: CORR
+  sys,cor30:
+    description: sys,cor30
+    treatment: MULT
+    type: CORR
+  sys,cor31:
+    description: sys,cor31
+    treatment: MULT
+    type: CORR
+  sys,cor32:
+    description: sys,cor32
+    treatment: MULT
+    type: CORR
+  sys,cor33:
+    description: sys,cor33
+    treatment: MULT
+    type: CORR
+  sys,cor34:
+    description: sys,cor34
+    treatment: MULT
+    type: CORR
+  sys,cor35:
+    description: sys,cor35
+    treatment: MULT
+    type: CORR
+  ATLASLUMI12:
+    description: ATLASLUMI12
+    treatment: MULT
+    type: SPECIAL
```

Comment:
I'm confused about the keyword `SPECIAL` here.
We have the CORR and UNCORR as variables that are correlated within a dataset, and then `type: <some name>`, variables that should be correlated between experiments.

For instance, in this case `ATLASLUMI12`. If you put `SPECIAL` it will be correlated with other stuff called `SPECIAL`. But you want to use `SPECIAL` also for the 13 TeV ones which would be wrong? (I think `ATLASLUMI12` is 8 TeV looking at where it appears.

Is there something I'm missing?

Reply:
> *and we cannot correlate between experiments here because the assumption of an experiment-diagonal covmat is hardcoded in a bunch of places. Luckily there's now a PR sort of dealing with that.

Just out of curiosity, which PR is dealing / dealt with that?